types/state/chain_manager.go

```diff
@@ -35,7 +35,7 @@ func (c *ChainManager) Height() int64 {
 
 // Record returns the record.
 func (c *ChainManager) Record() (*Object, error) {
-	data, err := c.state.dbMgr.Key(append(c.key, "Record")...).Get()
+	data, err := c.state.dbMgr.Key(append(c.key, "Record")).Get()
```

Comment:
`storage.ComputeKey` does not handle `[]interface{}`, so this will panic, I believe.